pkgs/os-specific/darwin/skhd/default.nix

```diff
@@ -1,31 +1,41 @@
-{ lib, stdenv, fetchFromGitHub, Carbon }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, Carbon
+, Cocoa
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "skhd";
-  version = "0.3.5";
+  version = "0.3.9";
 
   src = fetchFromGitHub {
     owner = "koekeishiya";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "0x099979kgpim18r0vi9vd821qnv0rl3rkj0nd1nx3wljxgf7mrg";
+    repo = "skhd";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-fnkWws/g4BdHKDRhqoCpdPFUavOHdk8R7h7H1dAdAYI=";
   };
 
-  buildInputs = [ Carbon ];
+  buildInputs = [
+    Carbon
+    Cocoa
+  ];
 
-  makeFlags = [ "BUILD_PATH=$(out)/bin" ];
+  makeFlags = [
+    "BUILD_PATH=$(out)/bin"
+  ];
 
   postInstall = ''
     mkdir -p $out/Library/LaunchDaemons
     cp ${./org.nixos.skhd.plist} $out/Library/LaunchDaemons/org.nixos.skhd.plist
     substituteInPlace $out/Library/LaunchDaemons/org.nixos.skhd.plist --subst-var out
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Simple hotkey daemon for macOS";
     homepage = "https://github.com/koekeishiya/skhd";
-    platforms = platforms.darwin;
-    maintainers = with maintainers; [ cmacrae lnl7 periklis ];
-    license = licenses.mit;
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ cmacrae lnl7 periklis khaneliman];
+    platforms = lib.platforms.darwin;
```

Comment:
I think there's an issue here

Reply:
what's that?

Reply:
I think you should update `meta.platforms` to support Linux and Darwin.

Reply:
It's macOS-only software.

Reply:
This is in the os-specific/darwin though, no?

Reply:
Oh... My bad... Reviewing too much PRs :)